src/routes/namespaces/[namespace]/workflows/[workflow]/[run]/index.svelte

```diff
@@ -21,22 +21,20 @@
 
 <script lang="ts">
   import PendingActivities from './_pending-activities.svelte';
-  import CodeBlock from '$lib/components/code-block.svelte';
   import Event from './_event.svelte';
 
   export let workflow: WorkflowExecution;
-  export let events: HistoryEvent[];
+  export let events: HistoryEventWithId[];
 
-  let format: EventFormat = 'grid';
-
-  $: pendingActivities = workflow?.pendingActivities;
+  let pendingActivities = workflow?.pendingActivities;
```

Comment:
This looks nullish here and we just don't have ts to yell at us. Is it okay for this to be null/undefined?

Reply:
It is. The component checks for its existence and its length.